src/controls/open-webui.js

```diff
@@ -0,0 +1,11 @@
+import {shell} from 'electron'
+import {apiAddrToUrl} from './utils'
+import {logger} from '../config'
+
+export default function openWebUI (ipfs, cb) {
```

Comment:
Also, always please use `callback` as the top level callback of a function and `cb` for internal ones (makes them easier to distinguish)